.appveyor.yml

```diff
@@ -6,6 +6,9 @@ install:
 shallow_clone: true
 build: off
 
+cache:
+  - "%LOCALAPPDATA%\\Yarn"
+
```

Comment:
It looks like our CircleCI setting also caches `node_modules` directory. Is there any reason you don't cache them?

Reply:
Oh, since that's `yarn`s' recommendation...

Reply:
We can add `node_modules` and make the cache depends on `yarn.lock` file, but I think this is good enough for yarn, since yarn is fast enough with (its own) cache...